core/templates/dev/head/pages/footer.html

```diff
@@ -42,6 +42,7 @@ <h4 translate="I18N_FOOTER_CONTRIBUTE_ALL_CAPS"></h4>
                   style="color: black;">
           </select>
         </ul>
+        <follow-button></follow-button>
```

Comment:
Let's call this social-buttons (and social_buttons_directive.html, SocialButtonsDirective, etc.).